src/routes/(site)/blog/[id]/blog-body.svelte

```diff
@@ -26,11 +26,11 @@
 >`;
     const svgCode = encodeURI(svg);
     for (let code of pre) {
-      let parentDiv = code.parentElement;
+      const parentDiv = code.parentElement;
       if (parentDiv) {
-        parentDiv!.style.position = 'relative';
+        parentDiv.style.position = 'relative';
```

Comment:
In CSS, we could write this out as...
```css
:global(div:has(pre)) {
  position: relative;
}
```

Reply:
done

Reply:
actually I don't think CSS applies it since the style got applied during the for loop through DOM manipulation

Reply:
> actually I don't think CSS applies it since the style got applied during the for loop through DOM manipulation

Even though the element is created in the for loop, I think there is still a way to properly select and style it using CSS. The CSS styles should apply in any DOM state.

Reply:
Daniel resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/629/commits/e8e104a4b920469abba97a437787121c17df6ac9>.